lambda-functions/cleanup-unused-images/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	log.SetLevel(log.DebugLevel)
+
+	lambda.Start(handler)
+}
+
+func handler() {
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String(os.Getenv("AWS_REGION"))},
+	)
+
+	svc := ec2.New(sess)
+	uniqueUsedImages, err := getUniqueUsedImages(svc)
+	err = deleteAMIs(svc,uniqueUsedImages)
+
+	fmt.Println(uniqueUsedImages)
```

Comment:
Is this for debug purposes? If so we should probably use `log.Info` instead of `fmt.Println` as we use the logger elsewhere already.

There are other occurrences of `fmt.Println` below that I would change as well.